wasm-engines/docker/Dockerfile

```diff
@@ -195,13 +195,34 @@ RUN update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 1
 RUN git clone https://github.com/wasmx/fizzy.git --single-branch --branch master
 RUN cd fizzy && mkdir build && cd build && cmake -DFIZZY_TESTING=ON .. && cmake --build .
 
+FROM ubuntu:19.10
```

Comment:
Could also use the`rust` image from dockerhub and that would remove the need for rustup.

Reply:
Oh, so this Dockerfile in particular is the old "monolithic" wasm engines docker image that should be deleted from this PR.